### ISSUE TITLE
Readmore doesn't work from time to time

### ISSUE THREAD

**Issue by rusrip** [jedfoster/Readmore.js#106]
Readmore sometimes doesn't work. There's no errors in console. 
Tested in Chrome 44.0.2403.30 beta-m, FF 34.0.5.

Initialization
```javascript
$('.desc_in').readmore({
		speed: 300,
		maxHeight: 280,
		moreLink: '<a href="#" class="desc_more openmore">читать далее</a>',
		lessLink: '<a href="#" class="desc_more closemore">Скрыть</a>'
	});
```
Height of this blocks is more than 280px

Examples
![readmore_not_ok](https://cloud.githubusercontent.com/assets/1313049/8176313/cb07ad22-142b-11e5-9ca1-2557ea39630d.png)
![readmore_ok](https://cloud.githubusercontent.com/assets/1313049/8176314/cb07fc1e-142b-11e5-888e-235ab57b9e11.png)

**Comment by lrobi2014** [comment]
Yes, I actually have notice this as well...any one have any suggestions?

**Comment by SantzDesign** [comment]
Same issue here, sometimes it doesn't work. Very odd.

**Comment by SantzDesign** [comment]
Hey, I think I found out what the bug is ( @jedfoster ).

When content exceeds the max-height by very little, it will expand the first time, but if you try to collapse it it won't work.

Hope this helps.

**Comment by lrobi2014** [comment]
@SantzDesign - That is interesting...thanks for that

**Comment by jedfoster** [comment]
Can someone provide a CodePen or jsbin where happens? There's not much I can do without a working example this issue.

**Comment by ssteigen** [comment]
I also ran into this issue. I tried to reproduce it in a codepen, but I wasn't able to.

In my case, the height of the item in the browser wasn't matching the collapse height value defined in the JS.

My `div.readmore` had `style="height:30px;"` but was appearing on the page with a height of `32px`.

I wasn't able to determine exactly where those 2 extra pixels were coming from, but removing margins and padding from `div.readmore` and instead adding it to the content within it seemed to fix the issue.

It looks like adding padding or margin to the `div.readmore` element can mess up the height calculation.
